seatunnel-server/seatunnel-app/src/main/java/org/apache/seatunnel/app/thirdparty/datasource/impl/Db2DataSourceConfigSwitcher.java

```diff
@@ -110,13 +110,15 @@ public Config mergeDatasourceConfig(
                 connectorConfig =
                         connectorConfig.withValue(QUERY_KEY, ConfigValueFactory.fromAnyRef(sql));
             } else if (pluginType.equals(PluginType.SINK)) {
-
-                List<String> tableFields = selectTableFields.getTableFields();
-
-                String sql = generateDb2(tableFields, databaseName, tableName);
-
-                connectorConfig =
-                        connectorConfig.withValue(QUERY_KEY, ConfigValueFactory.fromAnyRef(sql));
+                if (selectTableFields.isAll()) {
```

Comment:
I do not see any reference of generate_sink_sql change or use. an you explain the code change, how it is related to db2 upsert